lib/inquirer-prompt.js

```diff
@@ -18,11 +18,14 @@ module.exports = function prompt(questions, config) {
 	.reduce(function (newOpts, ans) {
 		return attachAnswers(newOpts, ans);
 	}, Object.assign(config, require('./utils/initial-config')))
-	.flatMap(newOpts => {
+	.map(function(newOpts) {
```

Comment:
Hmm, I thought the convention is to use `function` only for declarations, not for closures :/ Can you point me to the other usage?